leapp/cli/upgrade/__init__.py

```diff
@@ -156,10 +156,15 @@ def prepare_configuration(args):
     else:
         os.environ['LEAPP_EXPERIMENTAL'] = '0'
     os.environ['LEAPP_UNSUPPORTED'] = '0' if os.getenv('LEAPP_UNSUPPORTED', '0') == '0' else '1'
+    if args.no_rhsm or os.environ['LEAPP_NO_RHSM'] == '1':
+        os.environ['LEAPP_NO_RHSM'] = '1'
+    else:
+        os.environ['LEAPP_NO_RHSM'] = os.environ.get('LEAPP_DEVEL_SKIP_RHSM', '0')
     configuration = {
         'debug': os.getenv('LEAPP_DEBUG', '0'),
         'verbose': os.getenv('LEAPP_VERBOSE', '0'),
         'whitelist_experimental': args.whitelist_experimental or ()
+        'no_rhsm': os.environ['LEAPP_NO_RHSM']
```

Comment:
```suggestion
        'no_rhsm': os.getenv('LEAPP_NO_RHSM', '0')
```

Reply:
In this case we certainly set `LEAPP_NO_RHSM` to some value, so I don't see the point in calling `getenv`.

Reply:
true